src/xdocs/config_metrics.xml

```diff
@@ -216,6 +222,72 @@
     &lt;property name=&quot;max&quot; value=&quot;5&quot;/&gt;
 &lt;/module&gt;
         </source>
+
+        <p>
+          Override property <code>excludedPackages</code> to mark some packages as excluded.
+          Each member of <code>excludedPackages</code> should be a valid identifier:
+          <ul>
+            <li>
+              <code>java</code> - valid, exlcudes all <code>java.*</code> classes and packages.
```

Comment:
`exlcudes` -> `excludes`
This appears a few times in your changes. Please fix them all.

Also once https://github.com/checkstyle/checkstyle/pull/3924#discussion_r105546397 is changed, this will change as well.